curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/671f7767854d6806ac4fb5f4.md

```diff
@@ -0,0 +1,106 @@
+---
+id: 671f7767854d6806ac4fb5f4
+title: Task 101
+challengeType: 19
+dashedName: task-101
+---
+
+<!-- (Audio) Maria: I missed it. I was in a design sprint workshop. -->
+
+# --instructions--
+
+Listen to the audio and answer the question below.
+
+# --questions--
+
+## --text--
+
+Why could Maria not attend the cybersecurity workshop in the morning?
+
+## --answers--
+
+Because she wasn’t interested in the topic.
+
+### --feedback--
+
+Maria doesn’t mention a lack of interest; she was in another workshop.
+
+---
+
+Because she forgot about it.
+
+### --feedback--
+
+Maria says she missed it because she was attending the design sprint workshop, not because she forgot.
+
+---
+
+Because she was in a design sprint workshop.
+
+---
+
+Because it was canceled.
+
+### --feedback--
+
+Maria explains that she missed the workshop because she was in another one, not because it was canceled.
+
+## --video-solution--
+
+3
+
+# --explanation--
+
+The word `miss` means to fail to attend or participate in something. In this sentence, Maria missed the cybersecurity workshop because she was busy with another activity. Example: `I missed the meeting because I was stuck in traffic.` (this means that you spent too long in traffic and this caused a delay to get to work) 
+
+A `design sprint` is a time-boxed, collaborative process where a team works on solving a design problem or developing a prototype within a short period, usually a few days. Example: `The team completed a design sprint to test the new user interface concept.` (this means the team participated in this collaborative process to test the concept)  
+
+In this context, Maria couldn’t attend the cybersecurity workshop because she was actively participating in a design sprint workshop.
```

Comment:
```suggestion
`Miss` means to fail to attend or participate in something. For example: 

`I missed the meeting because I was stuck in traffic.` - This means that you spent too long in traffic and this caused a delay to get to work. 

A `design sprint` is a time-boxed, collaborative process where a team works on solving a design problem or developing a prototype within a short period, usually a few days. For example: 

`The team completed a design sprint to test the new user interface concept.` This means the team participated in this collaborative process to test the concept. 

In this context, Maria couldn’t attend the cybersecurity workshop because she was actively participating in a design sprint workshop.
```